koku/api/report/test/tests_queries.py

```diff
@@ -347,6 +347,47 @@ def test_set_operator_specified_tag_filters_or(self):
         self.assertIsInstance(output, QueryFilterCollection)
         assertSameQ(output.compose(), expected.compose())
 
+    def test_set_access_filter(self):
```

Comment:
Can you make this 2 tests? One for the list, the other for the non-list.

Reply:
Updated to be separate tests.